### DISCUSSION TITLE
Replace lerna with yarn workspaces

### DISCUSSION THREAD

**Discussion by kumaraditya303** [microsoft/pyright#1766] [Ideas]
The repo can use yarn workspaces which would be better than the current lerna monorepo

**Comment by erictraut** [comment]
No thank you. We're happy with the build configuration we have now. It was designed to work across pyright and pylance.

**Reply by jakebailey** [reply]
Yeah, yarn workspaces unfortunately do not nest well; I have ideas on how to make it work (because it would fix a few things), but at the moment we can't change this.
